app/views/layouts/_nav.html.erb

```diff
@@ -64,4 +64,17 @@
 			<% end %>
 		</ul>
 	</div>
+  <% if @deployment_notification.present?%>
+    <% if Time.zone.now >= @deployment_notification.display_time %>
+    <div class="notification-banner">
+      <div class="col-sm-3"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+        <span>Service Alert: Starting on</span></br>
+        <span><%= @deployment_notification.deployment_time.strftime("%a %b %d, %Y %I:%M%p") %> EST</span>
```

Comment:
Using ET (Eastern Time) would be more generic and accurate.  It abstracts away details of EST (Eastern Standard Time) and EDT (Eastern Daylight Time).

```suggestion
        <span><%= @deployment_notification.deployment_time.strftime("%a %b %d, %Y %I:%M%p") %> ET</span>
```